kythe/go/serving/pipeline/nodes/nodes.go

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-// Package nodes provides Beam transformations over *ppb.Nodes.
+// Package nodes provides Beam transformations over *scpb.Nodes.
```

Comment:
To the reader in http://godoc.org/, the `scpb` won't denote anything; I suggest "…over `kythe.proto.schema.Node` messages." or words to that effect.

Reply:
The package name does show up in the documentation: https://godoc.org/kythe.io/kythe/go/serving/pipeline/nodes#Kind.  I wanted to be consistent with the function types.

Reply:
That makes sense. SGTM.